hpack/table.py

```diff
@@ -0,0 +1,215 @@
+    (b':status'                     , b'304'          ),
+    (b':status'                     , b'400'          ),
+    (b':status'                     , b'404'          ),
+    (b':status'                     , b'500'          ),
+    (b'accept-charset'              , b''             ),
+    (b'accept-encoding'             , b'gzip, deflate'),
+    (b'accept-language'             , b''             ),
+    (b'accept-ranges'               , b''             ),
+    (b'accept'                      , b''             ),
+    (b'access-control-allow-origin' , b''             ),
+    (b'age'                         , b''             ),
+    (b'allow'                       , b''             ),
+    (b'authorization'               , b''             ),
+    (b'cache-control'               , b''             ),
+    (b'content-disposition'         , b''             ),
+    (b'content-encoding'            , b''             ),
+    (b'content-language'            , b''             ),
+    (b'content-length'              , b''             ),
+    (b'content-location'            , b''             ),
+    (b'content-range'               , b''             ),
+    (b'content-type'                , b''             ),
+    (b'cookie'                      , b''             ),
+    (b'date'                        , b''             ),
+    (b'etag'                        , b''             ),
+    (b'expect'                      , b''             ),
+    (b'expires'                     , b''             ),
+    (b'from'                        , b''             ),
+    (b'host'                        , b''             ),
+    (b'if-match'                    , b''             ),
+    (b'if-modified-since'           , b''             ),
+    (b'if-none-match'               , b''             ),
+    (b'if-range'                    , b''             ),
+    (b'if-unmodified-since'         , b''             ),
+    (b'last-modified'               , b''             ),
+    (b'link'                        , b''             ),
+    (b'location'                    , b''             ),
+    (b'max-forwards'                , b''             ),
+    (b'proxy-authenticate'          , b''             ),
+    (b'proxy-authorization'         , b''             ),
+    (b'range'                       , b''             ),
+    (b'referer'                     , b''             ),
+    (b'refresh'                     , b''             ),
+    (b'retry-after'                 , b''             ),
+    (b'server'                      , b''             ),
+    (b'set-cookie'                  , b''             ),
+    (b'strict-transport-security'   , b''             ),
+    (b'transfer-encoding'           , b''             ),
+    (b'user-agent'                  , b''             ),
+    (b'vary'                        , b''             ),
+    (b'via'                         , b''             ),
+    (b'www-authenticate'            , b''             ))
```

Comment:
We need to fix up the indentation here. This should be:

```python
STATIC_TABLE = (
    (b':authority'                  , b''             ),
    (b':method'                     , b'GET'          ),
)
```

Note the extra 4-deep indentation, and the closing paren being on its own line.